ionic_user_interface/src/views/ViewRoute.vue

```diff
@@ -8,6 +8,9 @@
           <div class="route-title">
             <b>{{ routeDetails.routeName }}</b>
           </div>
+          <div class="share-button-div icon-button margin-right" @click="sharePostHandler">
+            <ion-icon size="large" :icon="shareSocialOutline"></ion-icon>
+          </div>
           <VoteButton
```

Comment:
Oops, I think the votebutton doesn't have `cursor: pointer` on hover currently, can help me to add in?

Reply:
U added in the javascript

```
const css = '.vote-button-component:hover { background-color: #333333; cursor: pointer; }';
```